src/v2/Apps/Artwork/Components/ArtworkSidebar/__tests__/ArtworkSidebarMetadata.jest.tsx

```diff
@@ -150,6 +151,17 @@ describe("ArtworkSidebarMetadata", () => {
     })
   })
 
+  describe("for artwork with no size info", () => {
+    it("does not render any size info", async () => {
+      // @ts-expect-error PLEASE_FIX_ME_STRICT_NULL_CHECK_MIGRATION
```

Comment:
@jacobherrington - note that we shouldn't be using these flags. There's background info on this [here](https://github.com/artsy/force/blob/main/docs/best_practices.md#avoid-copying-and-try-to-fix--ts-expect-error-please_fix_me_strict_null_check_migration-flags)